src/bdk_cli.rs

```diff
@@ -117,8 +119,32 @@ where
         timeout: wallet_opts.electrum_opts.timeout,
     });
 
-    // Fall back to Electrum config if Esplora config isn't provided
-    let config = config_esplora.unwrap_or(config_electrum);
+    #[cfg(feature = "compact_filters")]
+    let compact_filter_config: Option<AnyBlockchainConfig> = {
+        let peerconfig = BitcoinPeerConfig {
+            address: wallet_opts.compactfilter_opts.address.clone(),
+            socks5: None,
+            socks5_credentials: None,
+        };
+
+        Some(AnyBlockchainConfig::CompactFilters(
+            CompactFiltersBlockchainConfig {
+                peers: vec![peerconfig],
```

Comment:
To make it sync faster you can connect multiple times to the same peer. I would add an extra option to `CompactFilterOpts` with the number of "cores" to use in parallel to sync. We could default to 2 or 4 which is reasonable on modern hardware

Reply:
Yes, that would make sense, but `BitcoinPeerConfig` isn't `Clone` so we cant just copy same config n number of times.

I was trying to see if instead of giving one address, we can provide multiple addresses in cli which then will be converted into `Vec<Peer>`. But doesn't seem to be a easy way to do that with `structopt`. 

If we make the `BitcoinPeerConfig` `Clone` in upstream. then we can add multiuple connection to a single peer here.

Maybe I can do that in a separate PR.

Reply:
This is solved by creating multiple `BitcoinPeerConfig` for each connection instead of cloning.